rtp/lib/membrane_demo_rtp/send_pipeline.ex

```diff
@@ -33,34 +36,35 @@ defmodule Membrane.Demo.RTP.SendPipeline do
           ]
         },
         video_realtimer: Membrane.Realtimer,
-        video_sink: %Membrane.Element.UDP.Sink{
+        video_sink: %Membrane.UDP.Sink{
```

Comment:
NIT: how about adding some aliases, so that not to have to write `Membrane.*` each time?